server/routes/v1/settings.js

```diff
@@ -36,5 +36,7 @@ router.post(
 	upload.single("logo", 1),
 	settings.updateLogo
 );
+router.get("/settings/labs", settings.getLabs)
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/logchimp/logchimp/pullRequest?prid=7771592)
```suggestion
router.get("/settings/labs", settings.getLabs);
```